Cargo.toml

```diff
@@ -33,11 +33,12 @@ harness = false
 travis-ci = { repository = "mcginty/snow", branch = "master" }
 
 [dependencies]
-arrayref = "0.3.5"
+arrayref = "0.3.6"
 rand_core = "0.5"
-subtle = "2.1"
+subtle = "2.2"
```

Comment:
Same, let's keep at 2.1 since we don't use any features in 2.2 specifically and there are no security issues.